--- a/src/server.ts	2018-01-03 23:57:56.778089717 +0900
+++ b/src/server.ts	2018-01-03 23:59:21.793848201 +0900
@@ -13,13 +13,6 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 */
-if (process.platform !== "win32") {
-    if (process.getuid() !== 0) {
-        console.error("root please.");
-        process.exit(1);
-    }
-}
-
 import { execSync } from "child_process";
 import _ from "./Mirakurun/_";
 import status from "./Mirakurun/status";
@@ -45,11 +38,6 @@
 setEnv("SERVICES_DB_PATH", "/usr/local/var/db/mirakurun/services.json");
 setEnv("PROGRAMS_DB_PATH", "/usr/local/var/db/mirakurun/programs.json");
 
-if (process.platform === "linux") {
-    execSync(`renice -n -10 -p ${ process.pid }`);
-    execSync(`ionice -c 1 -n 7 -p ${ process.pid }`);
-}
-
 _.config.server = config.loadServer();
 _.config.channels = config.loadChannels();
 _.config.tuners = config.loadTuners();
